client/index.html

```diff
@@ -356,25 +357,352 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<textarea class="input" name="userStyles" id="user-specified-css-input" placeholder="You can override any style with CSS here"></textarea>
 							</div>
-							<div class="col-sm-12">
-								<h2>About The Lounge</h2>
+						</div>
+					</div>
+				</div>
+
+				<div id="help" class="window">
```

Comment:
I'm wondering if it would be better to have this as a modal rather than a window? Not sure how you feel about that.